org-formation/600-access/_tasks.yaml

```diff
@@ -184,6 +184,7 @@ CostExplorerAccessPolicy:
     PolicyDocument: '{
```

Comment:
try using `PolicyDocument: >-`, like how it's done for [inlinePolicy in aws configs](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L811)

Reply:
@zaro0508 I was able to successfully test the JSON string as it is here with a manual sceptre deploy in itsandbox.